seguridad/views.py

```
@@ -7,25 +7,22 @@
-                    return JsonResponse(data, status=200)
-                else:
-                    data['error'] = 'Login Fallido!, usuario no esta habilitado'
+        cuenta = str(request.POST.get('')).strip()
+        user = authenticate(username=cuenta,password=request.POST['password'])
+        if user is not None:
+            if user.is_active:
+                login(request, user)
+                data['resp'] = True
+                data['user'] = user.username
+                return JsonResponse(data, status=200)
             else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)